connectivity/FEATURE_BLE/include/ble/gatt/GattCallbackParamTypes.h

```diff
@@ -443,8 +443,31 @@ struct GattDataSentCallbackParams {
 
 };
 
-using GattUpdatesEnabledCallbackParams        = GattDataSentCallbackParams;
-using GattUpdatesDisabledCallbackParams       = GattDataSentCallbackParams;
+/**
+ * Gatt Updates Enabled Attribute related events
+ *
+ * Used by `onUpdatesEnabled`
+ */
+struct GattUpdatesEnabledCallbackParams {
+
+    /**
+     * The handle of the connection that triggered the event.
+     */
+    ble::connection_handle_t connHandle;
+
+    /**
+     * The handle of the CCCD producing the event
+     */
+    GattAttribute::Handle_t attHandle;
+
+    /**
+     * The handle of the characteristic containing the CCCD  
+     */
+    GattAttribute::Handle_t charHandle;
```

Comment:
Can you indicate that this is the value handle. There's at least two attributes with different handles in a characteristic: The declaration attribute one and the value attribute.

Reply:
That's a good point: aaf9435f87b9294e82e206085cd475b1bd6c28dd. Thanks.